core/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 
 [package]
 name = "deno_core"
-version = "0.45.2"
+version = "0.45.3"
```

Comment:
Please bump to 0.46.0, 2610ceac20bc644c0b58bd8a95419405d6bfa3dd reduces the number of exports from deno_core and is not backwards compatible.